Yank/pipeline.py

```diff
@@ -338,6 +339,12 @@ def prepare_phase(positions_file_path, parameters_file_path, ligand_dsl, system_
             logger.info("gro: %s" % positions_file_path)
 
         positions_file = openmm.app.GromacsGroFile(positions_file_path)
+        if ('nonbonded_method' in system_options) and (system_options['nonbonded_method'] in _NONPERIODIC_NONBONDED_METHODS):
```

Comment:
The second part of the conditional is only evaluated if the first part evaluates to `True`.